ui/lib/kv/addon/routes/secret/details/edit.js

```diff
@@ -0,0 +1,39 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+import { withConfirmLeave } from 'core/decorators/confirm-leave';
+import { breadcrumbsForSecret } from 'kv/utils/kv-breadcrumbs';
+
+@withConfirmLeave('model.newVersion')
+export default class KvSecretEditRoute extends Route {
```

Comment:
following our other route class naming patterns this should be 
```suggestion
export default class KvSecretDetailsEditRoute extends Route {
```

Reply:
I got it.